deploy/crds/policy.open-cluster-management.io_policyautomations.yaml

```diff
@@ -55,6 +55,13 @@ spec:
                       job
                     minLength: 1
                     type: string
+                  policyViolationContextLimit:
+                    description: The maximum number of violating cluster contexts
+                      that will be provided to the Ansible job as extra variables.
+                      PolicyViolationContextLimit = 0 means no limitation and the
+                      default value is DefaultPolicyViolationContextLimit.
```

Comment:
If a customer looks at the CRD will they see what `DefaultPolicyViolationContextLimit` is set to?  I think you have to show that value.